athena-redshift/pom.xml

```diff
@@ -38,16 +38,11 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
-        <!-- https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-redshift -->
+        <!-- https://mvnrepository.com/artifact/software.amazon.awssdk/redshift -->
         <dependency>
-            <groupId>com.amazonaws</groupId>
-            <artifactId>aws-java-sdk-redshift</artifactId>
-            <version>${aws-sdk.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>com.amazonaws</groupId>
-            <artifactId>aws-java-sdk-redshiftserverless</artifactId>
-            <version>${aws-sdk.version}</version>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>redshift</artifactId>
```

Comment:
Does the v2 redshift include redshift serverless? Can you please test by deploying the connector and trying to connect to a redshift serverless database? This was added to fix a previous issue: https://github.com/awslabs/aws-athena-query-federation/pull/1945

Reply:
I have tested for Redshift serverless with public access. It can connect and fetch data.
As #1945 mentioned NoClassDefFoundError is thrown by some features. As I am not aware of that feature. Hence, I added serverless SDK v2 again.